code/modules/virus2/effect.dm

```diff
@@ -953,3 +953,41 @@ var/list/compatible_mobs = list(/mob/living/carbon/human)
 	stage = 1
 	activate(var/mob/living/carbon/mob,var/multiplier)
 		mob << "<span class = 'notice'> You feel optimistic!</span>"
+
+/datum/disease2/effect/anxiety
+	name = "Severe Anxiety"
+	stage = 1
+	activate(var/mob/living/carbon/mob,var/multiplier)
+		if(prob(20))
+			mob << "<span class='notice'>You feel anxious.</span>"
+		if(prob(15))
+			mob << "<span class='danger'>You feel butterflies in your stomach.</span>"
+		if(prob(10))
+			mob.visible_message("<span class='danger'>[mob] stumbles around in a panic.</span>", \
+											"<span class='userdanger'>You have a panic attack!</span>")
+			mob.confused += (rand(6,8))
+			mob.jitteriness += (rand(6,8))
+		if(prob(5))
+			mob.visible_message("<span class='danger'>[mob] coughs up butterflies!</span>", \
+												"<span class='userdanger'>You cough up butterflies!</span>")
+			new /mob/living/simple_animal/butterfly(mob.loc)
+			new /mob/living/simple_animal/butterfly(mob.loc)
+
+/datum/disease2/effect/lycan
+	name = "Lycancoughy"
```

Comment:
I cannot explain my brain the enitre thought pattern was this 'okay what if we made another virus...where you coughed up corgis..oh i know i can call it Lycancoughy cuase it sounds like lycanthrophy ' deeper in my brain "...wtf is your problem..."